docs/docs/References/Schemas.md

```diff
@@ -112,6 +112,9 @@ Here is a JSON object that conforms to the above schema:
 
 For much more detail on JSON schemas, including which keywords are available, what they mean, and where you can use them, see <https://json-schema.org/understanding-json-schema/>.
 
+!!! info
+    Be careful when using the JSON schema keyword `default`. This keyword is commonly misunderstood. Specifically, it does not mean "If the user does not provide a value, then this default value will be auto-substituted instead", as you might expect. In fact, in JSON schemas, `default` has no semantic meaning at all!
```

Comment:
Per json doc, "default" value substitution is left to the json processing tools - https://json-schema.org/understanding-json-schema/reference/generic.html. Does it make sense to get that specific and say, SDK nor the backend (DE code) will substitute values. And this might change in future.

Reply:
I added another sentence at the end that says that our current police will only use this for UI pre-filling. Let me know if this seems good to you.